### PR TITLE
New package: Enigma.VirtualBox version 11.00

### LINKED ISSUE #175447: [Package Request]: Enigma Virtual Box
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.enigmaprotector.com/en/aboutvb.html
Publisher: The Enigma Protector Developers Team
Package Name: Enigma Virtual Box
Description: Enigma Virtual Box enables application files and registry to be consolidated in a single executable file, without loss of efficiency and without virtualized files having to be extracted to the HDD.
Package Version: 11.00 Build 20240826
Installer URL: https://www.enigmaprotector.com/assets/files/enigmavb.exe

```[tasklist]
### Tasks
- [ ] https://github.com/microsoft/winget-pkgs/pull/175454
```

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.6.0 :rocket:

- Resolves #175447

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/175454)

### CODE REVIEW

File: manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: 湅杩慭嘠物畴污䈠硯_is1
+ReleaseDate: 2024-08-26
+AppsAndFeaturesEntries:
+- Publisher: 桔⁥湅杩慭倠潲整瑣牯䐠癥汥灯牥⁳敔浡
+  ProductCode: 湅杩慭嘠物畴污䈠硯_is1
```
Comment (KarbitsCode) [inline manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml:17]: Are these supposed to be chinese characters?

Comment (stephengillie) [reply manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml:17]: This doesn't seem to match the Registry:
image

**Comment by stephengillie** [comment]
Verify:
image

File: manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageVersion: '11.00'
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-08-26
+AppsAndFeaturesEntries:
```
Comment (Trenly) [inline manifests/e/Enigma/VirtualBox/11.00/Enigma.VirtualBox.installer.yaml:14]: This should be removed as it exactly matches the publisher specified in the default locale file

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**